pkg/operator/operator.go

```diff
@@ -829,6 +829,13 @@ func (o *Operator) Config(ctx context.Context, key string) (*manifests.Config, e
 	}
 	o.userWorkloadEnabled = *c.ClusterMonitoringConfiguration.UserWorkloadEnabled
 
+	err = c.LoadEnforcedBodySizeLimit(o.client, ctx)
+	if err != nil {
+		c.UseMinimalEnforcedBodySizeLimit()
+		klog.Warningf("Error loading enforced body size limit. Using minimal body size limit %s. Error: %v",
+			c.ClusterMonitoringConfiguration.PrometheusK8sConfig.EnforcedBodySizeLimit, err)
+	}
```

Comment:
I would disable the limit if an error is returned.

```suggestion
	if err := c.LoadEnforcedBodySizeLimit(o.client, ctx);  err != nil {
		c.ClusterMonitoringConfiguration.PrometheusK8sConfig.EnforcedBodySizeLimit = ""
		klog.Warningf("Error loading enforced body size limit, no body size limit will be enforced. Error: %v",
			err)
	}
```